### PR TITLE
Make group chat names censored

### LINKED ISSUE #111: Group chat names are not filtered
Screenshot 2022-10-28 9 51 19 AM

Screenshot 2022-10-28 10 07 27 AM

Should be marked as urgent, as this would allow people to create a group chat with the name of an inappropriate message, then add people, sending an inappropriate message to their inbox.

### PR DESCRIPTION
**Pull request by showierdata9978**
fixes #111

### CODE REVIEW

**Comment by JoshAtticus** [comment]
Change title to "Make group chat names censored"

**Review by tnix100 — APPROVED** [review]

File: meower.py [hunk 1]
```diff
@@ -1302,6 +1302,9 @@ def delete_post(self, client, val, listener_detected, listener_id):
                 # Post not found
                 self.returnCode(client = client, code = "IDNotFound", listener_detected = listener_detected, listener_id = listener_id)
         else:
+
+
+
```
Comment (MikeDev101) [inline meower.py:1307]: extra whitespace shouldn't be necessary

**Review by MikeDev101 — APPROVED** [review]

File: meower.py (hunk 1 above)
Comment (showierdata9978) [reply meower.py:1307]: nano shenanigans

**Comment by CST1229** [comment]
Was this pull request tested at all? It has a syntax error:
```
Traceback (most recent call last):
  File "C:\nonadprogrs\memeower\Meower-Server\main.py", line 5, in <module>
    from meower import Meower
  File "C:\nonadprogrs\memeower\Meower-Server\meower.py", line 1316
    val = self.supporter.wordfilter(val)
TabError: inconsistent use of tabs and spaces in indentation
```

**Comment by showierdata9978** [comment]
"random nano spacing"

**Comment by showierdata9978** [comment]
i noticed the spacing and fixed it

**Pull request merged**